systers_portal/templates/meetup/snippets/about_button.html

```diff
@@ -0,0 +1,3 @@
+<div class="sidebar-module mb40">   
+  <a class="btn btn-info btn-block mb15" href="{% url 'about_meetup_location' object.slug%}" role="button">About</a>
```

Comment:
The **mb15** has no effect. Please remove it. 
After `object.slug` add one space.